web-client/src/styles/custom.scss

```diff
@@ -1794,7 +1794,8 @@ button.change-scanner-button {
   margin-right: 4px;
 }
 
-.sealed-docket-entry {
+.sealed-docket-entry,
+.sealed-case-entry {
```

Comment:
There is an `iconSealed` class, but it doesn't do what I need (it is too opinionated). Rather than using that one or using `sealed-docket-entry` (which is poorly named in my context), I made a new, more appropriately named class.